modules/rosa-deleting-sts-iam-resources-account-wide.adoc

```diff
@@ -14,5 +14,5 @@ If you no longer need to install a ROSA with STS cluster by using {cluster-manag
 ====
 The account-wide IAM roles and policies might be used by other ROSA clusters in the same AWS account. You must only remove the resources if they are not required by other clusters.
 
-The {cluster-manager} and user IAM roles are required if you want to install and manage other ROSA clusters in the same AWS account by using {cluster-manager}. You must only remove the roles if you no longer need to install ROSA clusters in your account by using {cluster-manager}.
+The {cluster-manager} and user IAM roles are required if you want to install, manage, and delete other ROSA clusters in the same AWS account by using {cluster-manager}. You must only remove the roles if you no longer need to install ROSA clusters in your account by using {cluster-manager}. See the "Additional resources" for information on repairing your cluster if these roles are removed prior to deletion.
```

Comment:
Maybe add "section" after "Additional resources"

See the "Additional resources" section... ?